packages/redux/src/entities/types/draftOrder.types.ts

```diff
@@ -0,0 +1,3 @@
+import type { DraftOrder } from '@farfetch/blackout-client';
+
+export type DraftOrdersEntity = DraftOrder;
```

Comment:
In here we are creating an entity for a single draft order, so the name of the entity should be `DraftOrderEntity`